samples/hosting/docker/Core_7/Receiver/Program.cs

```diff
@@ -39,40 +36,8 @@ static IHostBuilder CreateHostBuilder(string[] args)
 
                     endpointConfiguration.UseSerialization<NewtonsoftSerializer>();
                     endpointConfiguration.EnableInstallers();
-                    endpointConfiguration.DefineCriticalErrorAction(OnCriticalError);
                     return endpointConfiguration;
                 });
         }
-
-        static async Task OnCriticalError(ICriticalErrorContext context)
```

Comment:
I think this needs to stay because we mention that the critical error should be overwritten even here https://docs.particular.net/nservicebus/hosting/docker-host/. Having then our sample not comply feels off or is it ok? Imagine if NSB is shutdown the container would just stay alive. At least `Environment.FailFast` seems to make sense so that the container can be restarted

Reply:
Ugh, missed that we recommend to override it, will add it back!